res/skins/Shade/mixer_panel.xml

```diff
@@ -558,6 +558,76 @@
                 </Connection>
               </PushButton>
 
+              <!--
+              **********************************************
+              Visual- Recording State
+              **********************************************
+              -->
+              <!--Button as visual indicator for recording state-->
+			  <PushButton>
+                <NumberStates>3</NumberStates>
+                <State>
+                  <Number>0</Number>
+                  <Pressed>skin:/btn/btn_record.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record.png</Unpressed>
+                </State>
+                  <State>
+                  <Number>1</Number>
+                  <Pressed>skin:/btn/btn_record_overdown.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_overdown.png</Unpressed>
+                </State>
+                <State>
+                  <Number>2</Number>
+                  <Pressed>skin:/btn/btn_record_over.png</Pressed>
+                  <Unpressed>skin:/btn/btn_record_over.png</Unpressed>
+                </State>
+                <Pos>214,18</Pos>
+                <Connection>
+                  <ConfigKey>[Recording],status</ConfigKey>
+                </Connection>
+              </PushButton>
+
+               <!--
+              **********************************************
+              Text- Recording Duration
+              **********************************************
+              -->
+              <RecordingDuration>
+				<Style>
+                  QLabel {
+                    font-size: 9px;
+                    background-color: transparent;
+                    color: #191F24;
+                    text-align:center;
+                  }
+                </Style>
+				<ObjectName>RecDuration</ObjectName>
+				<TooltipId>recording_duration</TooltipId>
+				<Pos>216,18</Pos>
+				<Size>26f,14f</Size>
```

Comment:
the font/background contrast is too low. please use the dark font of the buttons.

what happens when the recording exceeds 99:59?
maybe the duration can be placed above the REC button (align: center?) where there is more space?

Reply:
Nitpicker!  :laughing:  
Yeah, I am aware of this problem, but was also wondering if that would ever happen in reality. 
Anyhow, I understand that this could be an issue and will try to find a better solution, but preferably without breaking the symmetry. So having the duration above the REC button is not my favorite option. 
I will try to position (just) the REC duration near (above or below) Time / On Air, leave the REC button as it is in my current solution and hope that this does not feel to 'out of place'.
Otherwise I may as well consider to discard the symmetry completely and go with the solution suggested in b1924279 - place the REC duration above (or below) the Time and make the Recording toggle-able by clicking on it. (similar to the On Air 'button').

The format of the REC duration seems, unlike the Time format, to not be configurable? (e.g having --:--:-- instead of --:--) Is this correct?

Reply:
a 2hour set is rather common. right now I can't look up or test the format is hh:mm:ss but I think it is.
the symmetry is based on the knobs and the two buttons on the left and right, the asymmetric label is absolutely okay. 
for now I very much recommend to leave the button as it is (discard the invisible layer) and move the duration above, if you would like to get this changes into 2.3
after all it is just a small detail which can be changed any time.
the current solution is an improvement already.

if you want to propose alternative designs I recommend making mockups using screenshots, for example with Gimp, instead of wasting time trying to implement designs immediately.